src/nyc_trees/gulpfile.js

```diff
@@ -153,6 +153,7 @@ function browserifyTask(bundler) {
         })
         .bundle()
         .on('error', gutil.log.bind(gutil, 'Browserify Error'))
+        .on('error', process.exit.bind(process, 1))
```

Comment:
@kdeloach The `debug` flags does not necessarily imply that `watchify` is running.  Though gating on `debug` would at least prevent Jenkins builds from breaking.